concepticondata/conceptlists/Monnier-2014-1031.tsv

```diff
@@ -0,0 +1,1032 @@
+Monnier-2014-1031-395	395	face 	face	FACE	1560	NOUN			5.05405405405405	3.64864864864865	5.17647058823529	3.76470588235294	4.95	3.55	4	3	1	124.33	262.16	3.9
+Monnier-2014-1031-396	396	facilité	ease			NOUN			6.88311688311688	3.88181818181818	5.44444444444444	3.66666666666667	7.07352941176471	3.91029411764706	8	8	4	1.47	11.55	2.45
+Monnier-2014-1031-397	397	faculté	faculty			NOUN			6.48101265822785	4.63291139240506	6.125	3.375	6.52112676056338	4.77464788732394	7	7	3	5.93	13.85	2.73
+Monnier-2014-1031-398	398	fainéant	lazy			NOUN			4.78441558441558	4.44805194805195	5	3.75	4.74	4.57692307692308	8	5	3	1.59	0.81	2.37
+Monnier-2014-1031-399	399	famille	family	FAMILY	194	NOUN			6.97297297297297	6	6.82352941176471	5.41176470588235	7.1	6.5	7	5	2	357.75	241.69	5.3
+Monnier-2014-1031-400	400	fanfare	brass band			NOUN			6.10810810810811	5.40540540540541	6.11764705882353	5.29411764705882	6.1	5.5	7	5	2	3.8	3.72	4.45
+Monnier-2014-1031-401	401	fantasme	fantasy			NOUN			7.54545454545455	6.87012987012987	7.77777777777778	6.88888888888889	7.51470588235294	6.86764705882353	8	6	2	4.64	1.49	
+Monnier-2014-1031-402	402	fantôme	ghost	GHOST	1175	NOUN			4.5945945945946	5.2972972972973	5.47058823529412	4.23529411764706	3.85	6.2	7	5	2	29.71	20.74	5.98
+Monnier-2014-1031-403	403	faon 	fawn			NOUN	B	133	6.18918918918919	3.78378378378378	6.35294117647059	4.35294117647059	6.05	3.3	4	2	1	0.25	0.54	5
+Monnier-2014-1031-404	404	fatigue	tiredness			NOUN			2.52	3.1	2.94444444444444	4.05555555555556	2.28125	2.5625	7	5	2	8.7	65.81	4.47
+Monnier-2014-1031-405	405	faute	mistake	MISTAKE	1293	NOUN			2.94594594594595	5.16216216216216	3.23529411764706	6.05882352941176	2.7	4.4	5	3	1	163.19	81.08	3.27
+Monnier-2014-1031-406	406	faveur	favor			NOUN			6.50632911392405	4.45569620253165	6.25	3.625	6.53521126760563	4.54929577464789	6	5	2	31.09	27.64	2.83
+Monnier-2014-1031-407	407	fée 	fairy	ELF OR FAIRY	3206	NOUN	B	134	6.81081081081081	4.75675675675676	6.64705882352941	4.76470588235294	6.95	4.75	3	2	1	8.3	6.62	6.3
+Monnier-2014-1031-408	408	féminine	feminine			ADJ			7.5974025974026	5.51948051948052	6.75	5.91666666666667	7.75384615384615	5.44615384615385	8	7	3	6.5	13.38	
+Monnier-2014-1031-409	409	femme 	woman	WOMAN	962	NOUN	B	135	7.39473684210526	5.59473684210526	8.29411764705882	7	6.66666666666667	4.45714285714286	5	3	1	806.57	680.2	6.2
+Monnier-2014-1031-410	410	fenêtre	window	WINDOW	1375	NOUN	AF	257	5.74	3.22	5.38888888888889	3.61111111111111	5.9375	3	7	6	2	70.2	199.39	6.9
+Monnier-2014-1031-411	411	fer	iron	IRON	621	NOUN			4.78947368421053	4.05263157894737	5.05882352941176	4.70588235294118	4.57142857142857	3.52380952380952	3	3	1	33.65	106.28	5.52
+Monnier-2014-1031-412	412	fermeture	closure			NOUN			3.62162162162162	3.18918918918919	3.58823529411765	3.11764705882353	3.65	3.25	9	8	3	11.06	14.86	4.11
+Monnier-2014-1031-413	413	féroce	ferocius			ADJ			3.32467532467532
```

Comment:
French 'longue' should refer to time, I suppose. Then it should rather be linked to 418 FOR A LONG TIME.